test/com/google/javascript/jscomp/TypeCheckTest.java

```diff
@@ -43,7 +43,6 @@
 
 public final class TypeCheckTest extends CompilerTypeTestCase {
 
-  private static final Joiner LINE_JOINER = Joiner.on("\n");
   private static final String SUGGESTION_CLASS =
```

Comment:
Is this `SUGGESTION_CLASS` used internally? It doesn't seem to be used externally so I can remove it as well.